docs/guide/ordering.md

```diff
@@ -33,6 +39,181 @@ input FruitOrder {
 }
 ```
 
+## Custom order methods
+
+You can define custom order method by defining your own resolver.
+
+```{.python title=types.py}
+@strawberry_django.order(models.Fruit)
+class FruitOrder:
+    name: auto
+
+    @strawberry_django.order_field
+    def discovered_by(self, value: bool, prefix: str) -> list[str]:
+        if not value:
+            return []
+        return [f"{prefix}discover_by__name", f"{prefix}name"]
+
+    @strawberry_django.order_field
+    def orderer_number(
```

Comment:
orderer_number -> order_number
maybe wrong typo?

Reply:
You are completely correct, good catch. Fixed.